docs/site_navigation/redirects.md

```diff
@@ -7,27 +7,29 @@ nav_order: 3
 
 # Redirects
 
-The URL redirects are managed in **My Site > Redirects**, both Creators and Developers have access.
+The URL redirects are managed in **My Site > Redirects**, by anyone with the 'Create and edit site pages' permission. Redirects can be chained and can be set to unpublished pages (in that case users not logged in the Easol platform will be rerirected to a login page).
```

Comment:
Hey @scsx just spotted a typo here _rerirected_
I'm not super clear on what behaviour this is describing. Is it that it is possible to set up redirects to an unpublished page, but that any customers / users who aren't company admins will be redirected to a log in page?

Reply:
Hey @paulina-a yes, is it unnecessary or unclear?